src/App.tsx

```diff
@@ -8,37 +8,118 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import {
+  getUsers,
+  getPosts,
+  getComments,
+  addComment,
+  deleteComment,
+} from './api/users';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { NewComment } from './types/NewComment';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [selectedUser, setSelectedUser] = useState<User | undefined>(undefined);
+  const [userIsLoading, setUserIsLoading] = useState<boolean>(false);
+  const [posts, setPosts] = useState<Post[]>([]);
+  const [postsLoadingError, setPostsLoadingError] = useState<boolean>(false);
+  const [selectedPost, setSelectedPost] = useState<Post | undefined>(undefined);
+  const [comments, setComments] = useState<Comment[]>([]);
+  const [commentsError, setCommentsError] = useState<boolean>(false);
+  const [commentsAreLoading, setCommentsAreLoading] = useState(false);
```

Comment:
```suggestion
  const [isUserLoading, setIsUserLoading] = useState<boolean>(false);
  const [posts, setPosts] = useState<Post[]>([]);
  const [isPostsLoadingError, setIsPostsLoadingError] = useState<boolean>(false);
  const [selectedPost, setSelectedPost] = useState<Post | undefined>(undefined);
  const [comments, setComments] = useState<Comment[]>([]);
  const [isCommentsError, setIsCommentsError] = useState<boolean>(false);
  const [areCommentsLoading, setAreCommentsLoading] = useState(false);
```
![image](https://user-images.githubusercontent.com/96409563/201884281-271ecbe4-7a2f-4abe-a89a-7e6e7b88a6b9.png)